retrofit-converter-jspoon/src/main/java/pl/droidsonroids/retrofit2/JspoonResponseBodyConverter.java

```diff
@@ -1,19 +1,27 @@
 package pl.droidsonroids.retrofit2;
 
 import java.io.IOException;
+import java.io.InputStream;
+
+import okhttp3.HttpUrl;
 import okhttp3.ResponseBody;
 import pl.droidsonroids.jspoon.HtmlAdapter;
 import retrofit2.Converter;
 
 class JspoonResponseBodyConverter<T> implements Converter<ResponseBody, T> {
-    private HtmlAdapter<T> htmlAdapter;
 
-    JspoonResponseBodyConverter(HtmlAdapter<T> htmlAdapter) {
+    private final HttpUrl httpUrl;
+    private final HtmlAdapter<T> htmlAdapter;
+
+    JspoonResponseBodyConverter( HttpUrl httpUrl, HtmlAdapter<T> htmlAdapter ) {
```

Comment:
Please use consistent spacing. Remove spaces before first and after last parameter in this case.